README.md

```diff
@@ -48,6 +50,22 @@ Note: The latest release of acs can also be installed locally with the following
 cargo install autoclockspeed
 ```
 
+## Tested Devices
+Auto clock speed has been tested to work on the following devices. If you have a device that is not listed please submit a pull request.
+
+| Device Name | Functional |
+| ----------- | ---------- |
+| Dell XPS 13 9360 | Working |
+| Dell Latitude 7480 | Working |
+| Steam Deck | Working |
+| Thinkpad T400 | Working |
+| Thinkpad X230 | Working |
+| Thinkpad W540 | Working |
+| ThinkPad X1 Extreme Gen 1 | Working |
+| Thinkpad P1 Gen 4 (Intel Core) | Working |
+| Thinkpad P14 Gen 2 (AMD) | Working |
```

Comment:
Might wanna set this to partially working though (idt we figured out the clockspeed issue yet)

Reply:
True